crates/forge-runner/src/contracts_data.rs

```diff
@@ -0,0 +1,32 @@
+use anyhow::Result;
+use bimap::BiMap;
+use cheatnet::runtime_extensions::forge_runtime_extension::cheatcodes::declare::get_class_hash;
+use std::collections::HashMap;
+
+use rayon::iter::{IntoParallelRefIterator, ParallelIterator};
+use scarb_api::StarknetContractArtifacts;
+use starknet_api::core::{ClassHash, EntryPointSelector};
+
+#[derive(Debug, Clone)]
+pub struct ContractsData {
+    pub contracts: HashMap<String, StarknetContractArtifacts>,
+    pub class_hashes: BiMap<String, ClassHash>,
```

Comment:
We're using `get_by_right`, I guess it's not the case for 2 different named contracts to have the same class hash?

Reply:
Yup, it is not, bimap is here so it's easier to reuse this struct in declare -  see #1719